spec/features/user_visits_login_page_spec.rb

```diff
@@ -50,4 +50,35 @@
     end
   end
 
+  context "they can't create a new user" do
+    it "with a password that already exists" do
```

Comment:
should be "with a username that already exists"

you're allowed to use the same pword as someone else